README.md

```diff
@@ -259,12 +259,13 @@ rm "$PREFIX/bin/ani-cli"
 - sed
 - awk
 - curl
+- wget
```

Comment:
Why do we have both wget and curl now?
Looks kinda stupid, don't they have feature parity?

Reply:
Seems like @CoolnsX prefers `wget`
Only one occurence of `curl` is remaining, we can get rid of that

Reply:
okay i m on it

Reply:
also i need to do some fixing.. so

Reply:
Wget was not a preference, in fact curl is preferred by most of us but it misbehaved on windows

Reply:
> Wget was not a preference, in fact curl is preferred by most of us but it misbehaved on windows

Good to know. At least we're making it more portable I guess